components/modal/index.jsx

```diff
@@ -79,11 +79,11 @@ const propTypes = {
 	 */
 	directional: PropTypes.bool,
 	/**
-	 * If true, Modals can be dismissed by clicking on the close icon or pressing esc key.
+	 * If true, Modals cannot be dismissed by clicking on the close icon or pressing esc key.
 	 */
-	dismissible: PropTypes.bool,
```

Comment:
Removing this will break consumers of `dismissible`. We need to deprecate `dismissible` and the new prop `disableClose`. Take a look at `check-props.js` to see how it's done.

Reply:
I will patch this by evening :)

Reply:
I am adding the following into `check-props.js` to show up a message -
```
deprecatedProperty(
			COMPONENT,
			props.dismissible,
			'dismissible',
			"disableClose",
			'The prop `dismissible` has been deprecated, please use `disableClose` instead.'
		);
```
@davidlygagnon would that be enough? Thank you :)